src/test/java/com/tekenable/trialdirect/config/AuditConfig.java

```diff
@@ -0,0 +1,27 @@
+package com.tekenable.trialdirect.config;
+
+import com.tekenable.config.AuditConfigurator;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.datasource.SimpleDriverDataSource;
+
+import javax.sql.DataSource;
+import java.sql.SQLException;
+
+@Configuration
+public class AuditConfig {
+
+    @Bean(name = "dataSource")
+    public DataSource dataSource() throws SQLException {
+        final SimpleDriverDataSource dataSource = new SimpleDriverDataSource();
+        dataSource.setDriver(new com.mysql.jdbc.Driver());
+        dataSource.setUrl("jdbc:mysql://localhost:3306/trialdirect?zeroDateTimeBehavior=convertToNull");
```

Comment:
Could you create an auditConfig.properties.template file with the string variables used in the datasource creation.
Autowire a Properties object into the class here looking for "auditConfig.properties" minus the template.
If it doesn't exist throw an exception instructing the user to copy'n'paste the template and insert their own specific values.